packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -213,8 +214,8 @@ export interface SomethingSignature {
 
 export class Something extends Component<SomethingSignature> {
   /** */
-  // @ts-expect-error @rehearsal TODO TS7019: Rest parameter 'args' implicitly has an 'any[]' type.
-  reset(...args): void {
+  // @ts-expect-error @rehearsal TODO TS7050: Function 'reset' lacks a return-type annotation.
+  reset(...args) {
```

Comment:
This isn't ideal, because we had `void`

The `...args` fails in `.gts` file because of a missing `symbol` on the parameter node. 

We could backstop this with bring back our inlay hints logic when this fails the `try..catch`